tests/nosetests/pyanaconda_tests/module_source_hmc_test.py

```diff
@@ -40,6 +40,11 @@ def type_test(self):
         """Test the type of SE/HMC."""
         self.assertEqual(SOURCE_TYPE_HMC, self.interface.Type)
 
+    @patch("pyanaconda.modules.payloads.source.hmc.hmc._", side_effect=lambda x: x)
```

Comment:
You don't have to patch the `_` functions. We set the locale in unit tests (see https://github.com/rhinstaller/anaconda/commit/50db2816e8943512ec2cfc6b6deae588ba88e057).

Reply:
Perfect, thank you!